discourse.Dockerfile

```diff
@@ -46,16 +45,11 @@ RUN ln -s /usr/share/zoneinfo/UTC /etc/localtime \
     tzdata \
     ubuntu-dev-tools \
     zlib1g-dev \
-    && rm /var/lib/apt/lists/* \
-# Older versions of the uglifyjs.terser package install a uglifyjs.terser
-# command but not the terser command, terser command exists in $PATH is
-# vital to trigger js assets compression using node. Manually create the
-# terser command if it does not exist. Please remove this line if the
-# base image is >= 22.04. Also, please consider removing the node-uglify
-# when upgrading to discourse >= 2.8.0, since the node-uglify is not
-# required to trigger node js assets compression, only terser will do fine.
-    && which terser || ln -s $(which uglifyjs.terser) /usr/local/bin/terser \
-# Run build process.
+    && curl --silent --location https://deb.nodesource.com/setup_18.x | bash - \
+    && apt-get update \
+    && apt-get install -y nodejs \
```

Comment:
Why not installing nodejs with the packages ?

Reply:
Which packages are you referring to?

Reply:
The node version in the default package list doesn't match the one we need. Additionally, I need curl, which is not installed by default.

Reply:
Ok got it
1) Install curl
2) Add node repo
3) Install node

I didn't realise the `curl | bash` before was adding a new deb repo